src/Labels.tsx

```diff
@@ -2,9 +2,18 @@ import {defined} from './Base';
 import {createLabel, fetchLabels} from './Gapi';
 
 const TEAMAIL_BASE_LABEL = 'tm';
+const MKTIME_BASE_LABEL = 'mktime';
+const PRIORITY_LABEL_NAME = `${MKTIME_BASE_LABEL}/priority`;
 
 export const LabelName = {
   keep: `${TEAMAIL_BASE_LABEL}/keep`,
+  bookmark: `${PRIORITY_LABEL_NAME}/Bookmark`,
+  pin: `${PRIORITY_LABEL_NAME}/Pin`,
+  emptyDaily: `${PRIORITY_LABEL_NAME}/Empty-daily`,
+  urgent: `${PRIORITY_LABEL_NAME}/Urgent`,
+  backlog: `${PRIORITY_LABEL_NAME}/Backlog`,
+  stuck: `${PRIORITY_LABEL_NAME}/Stuck`,
+  softMute: `${MKTIME_BASE_LABEL}/softmute`,
```

Comment:
Whoops. Looking below, I see you called the getter for id "getId", and in the patch I just landed, I called it "id".

We should probably be consistent, at least eventually. Happy to rename Message::id() and Thread::id() if that's the direction we want to go.

Reply:
Oof. Yeah. I'm always torn about this specific style tradeoff. Each solution leaves so much to be desired. So I end up just being inconsistent. I'm happy to go with whatever you prefer.